version.php

```diff
@@ -29,7 +29,7 @@
 
 defined('MOODLE_INTERNAL') || die();
 
-$module->version   = 2014101300;      // The current module version (Date: YYYYMMDDXX)
+$module->version   = 2014101302;      // The current module version (Date: YYYYMMDDXX)
```

Comment:
Please update this to 2015030200 or similar to reflect the later date this was updated on.